netket/models/equivariant.py

```diff
@@ -231,9 +237,10 @@ def __call__(self, x):
 
         x = self.output_activation(x)
 
-        x = logsumexp(
-            x, axis=(-2, -1), b=jnp.expand_dims(jnp.asarray(self.characters), (0, 1))
-        )
+        if self.ensure_cplx:
+            x = logsumexp_cplx(x, axis=(-2, -1), b=jnp.asarray(self.characters))
+        else:
+            x = logsumexp(x, axis=(-2, -1), b=jnp.asarray(self.characters))
```

Comment:
Can you add a test somewhere (1, not the hundreds that come from adding the combination...) testing `ensure_cplx=False`? It seems it's not covered.